vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -783,6 +799,33 @@ void BatGetMedia::onGetChannelIdFromUserPage(
   }
 }
 
+void BatGetMedia::onFetchDataFromNonEmbeddable(
+    uint64_t windowId,
+    const ledger::VisitData& visit_data,
+    const std::string& providerType,
+    const uint64_t& duration,
+    const std::string& media_key,
+    const std::string& mediaURL,
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers) {
+  if (response_status_code != 200) {
+    onMediaActivityError(visit_data, providerType, windowId);
+    return;
+  }
+
+  if (providerType == YOUTUBE_MEDIA_TYPE) {
+    std::string publisherName = parsePublisherName(response);
+    std::string publisher_url =
+        getPublisherUrl(parseChannelId(response), providerType);
+    fetchDataFromUrl(publisher_url,
```

Comment:
based on our call, let's remove this additional call and get everything from the first one